libs/models/src/lib/state/log-message.ts

```diff
@@ -0,0 +1,28 @@
+export enum AppErrorCode {
```

Comment:
Please add consistent spacing between the type definitions

Reply:
Add consistent spacing 755e6524b5cc463844463d50060be3604003a6a2